### DISCUSSION TITLE
Exit of code due to system crash running Contact Model

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#18718] [Q&A Modules: General] [answered]
I have developed a contact model, and I get the following error when I run the code:
Reiniting because of ghosting ....                                                         [  5.17 s]
Crashed
Crashed: Exit code: 11

The geometry and meshing appear to be well developed. I am thinking that the error may be due to the time stepping and the domain size (modeled moving and colliding components on the order of 1mm) of the objects involved. I used the indentation example model as a basis for my model, maintaining all the defined functions as is. The scale of that 2D model and mesh are much larger that my 2D model. Any ideas on what may be causing the crash?

**Comment by GiudGiud** [comment]
Hello

Could you be running out of memory?
If not could you please compile in debug mode (`METHOD=dbg make`) then run in a debugger to get a backtrace for the crash
(shorthand: `gdb --ex run --ex bt --args executable -i input_file.i`)

**Comment by TLWise** [comment]
An error with the mesh file of the exodus code caused the crash. When certain node sets were selected in peacock, it would cause a crash of the software, which in turn would have to be restarted.

**Reply by GiudGiud** [reply]
do you consistently get that crash? Are you comfortable providing us with a backtrace for it?

**Comment by TLWise** [comment, marked as answer]
I realized the mesh file was causing the crash related to this problem. Thank you for your continued support.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
